capstone/src/main/webapp/profile-loader.js

```diff
@@ -43,6 +43,10 @@ function getStudentInfo() {
 function createClubElement(text) {
   var liElement = document.createElement('li');
   liElement.innerText = text + '  ';
-  liElement.innerHTML += '<button>Leave</button>';
+
+  // Create leave button and set value to its respective club
+  liElement.innerHTML += '<button name="club" value="'
+    + text
+    + '" formmethod="POST">Leave</button>';
```

Comment:
Do you need to set both .innerText and .innerHTML?

Reply:
I don't think so? I just tried it without the .innerText and moved the .innerText portion to .innerHTML and it works as well!

Reply:
https://medium.com/better-programming/whats-best-innertext-vs-innerhtml-vs-textcontent-903ebc43a3fc

For clarity on what the difference is (if you didn't already know), had to look it up myself. 

Might be worth asking Sean if this is best practice for adding this stuff. I know we've already been doing, but at the very least would be good to know for sure! So you'll learn more applicable JS for future web development!

Reply:
Will do! Thanks for sending the link as well :)

Reply:
Sean just confirmed that .innerHTML is fine for updating a DOM node :D

Reply:
Nice! Thanks for confirming :)